go/vumitools/middleware.py

```diff
@@ -607,6 +608,9 @@ class ConversationMetricsMiddlewareConfig(BaseMiddleware.CONFIG_CLASS):
 
     redis_manager = ConfigDict(
         "Redis configuration parameters", default={}, static=True)
+    local_store_reset_timer = ConfigInt(
+        "Time in seconds to store conversations keys for locally.",
+        default=1800, static=True)
```

Comment:
30 minutes is probably too long given our current "publish metrics every hour", so I think we should change this default to 5 minutes.